onnx2kerastl/reshape_layers.py

```diff
@@ -264,7 +266,7 @@ def target_layer(x):
                         if input_0.shape.rank == 1:
                             input_0 = tf.expand_dims(input_0, 0)
                         logger.debug('The first argument is Keras/tf layer. Apply keras.Flatten.')
-                        flatten = keras.layers.Flatten(name=keras_name)
+                        flatten = keras.layers.Reshape(target_shape=input_1[1:], name=keras_name)
```

Comment:
Just So'd I know:

`keras.layers.Reshape(-1)(in_tensor) ` != `keras.layers.Flatten(in_tensor) ` 

These two order the elements in a different way?